src/System.Private.ServiceModel/tools/setupfiles/SetupWCFTestService.cmd

```diff
@@ -21,7 +21,7 @@ if '%BridgeAllowRemote%' neq '' (
    set _bridgeAllowRemoteArg=-allowRemote %BridgeAllowRemote%
 )
 
-echo Executing: start powershell -ExecutionPolicy Bypass -File ..\test\Bridge\bin\ensureBridge.ps1 %_bridgeHostArg% %_bridgePortArg% %_bridgeAllowRemoteArg%
+echo Executing: start powershell -ExecutionPolicy Bypass -File ..\test\Bridge\bin\ensureBridge.ps1 %_bridgeHostArg% %_bridgePortArg% %_bridgeAllowRemoteArg% %2 %3 %4 %5 %6 %7
 
-start powershell -ExecutionPolicy Bypass -File ..\test\Bridge\bin\ensureBridge.ps1 %_bridgeHostArg% %_bridgePortArg% %_bridgeAllowRemoteArg%
+start powershell -ExecutionPolicy Bypass -File ..\test\Bridge\bin\ensureBridge.ps1 %_bridgeHostArg% %_bridgePortArg% %_bridgeAllowRemoteArg% %2 %3 %4 %5 %6 %7
```

Comment:
I think I will just remove the need to pass the working folder through %1. But remember, I intend to remove as many of these CMD --> VBS --> PS --> CMD --> EXE transitions as I can shortly.